.gitlab/check-each-crate.py

```diff
@@ -55,7 +55,7 @@
 
 	print(f"Checking {crates[crate][0]}", file=sys.stderr)
 
-	res = subprocess.run(["cargo", "check", "--locked"], cwd = crates[crate][1])
+	res = subprocess.run(["forklift", "cargo", "check", "--locked"], cwd = crates[crate][1])
```

Comment:
Does the forklift have a `--quiet` option or something?  
I dont know if the noise here comes from forklift maybe https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/5143188 but its difficult to see why the job failed.

Reply:
errors start at line [3708](https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/5143188#L3708) with ```error[E0599]: the function or associated item `new` exists for struct `BenchmarkingState<Block<ExtrinsicWrapper<u64>>>`, but its trait bounds were not satisfied```

but yeah, that output seems redundant now, I'll fix it

FYI there are no flags as settings provided by config file. But it is possible to set vars by environment
`FORKLIFT_general_logLevel="warn"`

Reply:
fixed, from now on forklift will only write simple startup message and status report at the end (and default logLevel is now `warn`)